packages/commonwealth/client/scripts/views/components/sidebar/CommunitySection/CommunitySection.tsx

```diff
@@ -43,7 +44,15 @@ export const CommunitySection = ({ showSkeleton }: CommunitySectionProps) => {
           address={activeAccount?.address}
         />
       )}
-      {showAdmin && <AdminSection />}
+
+      {featureFlags.newCreateCommunity && <CreateCommunityButton />}
+
+      {showAdmin && (
+        <>
+          <CWDivider />
```

Comment:
i think the divider should be above with the feature flag condition.

Reply:
These are all 3 cases that need to be handled:
1. joined - admin
2. joined - not admin
3. not joined
4. not logged in

All of them work fine with this code.

<img width="324" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/a3b16190-6bcc-4e9d-b2db-14aa5a417f44">

<img width="293" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/3553c502-0051-40d7-a5ec-52b1590641d8">

<img width="296" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/d4cd7cfd-ee62-417e-a1d4-20b58c5c857d">

<img width="303" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/14819225/96d284d3-64c3-4db3-8a29-6e8e8919bc8e">

Reply:
ah got it. Thanks!